### PR TITLE
Phase2-gex82 Add 2 new phase2 scenarios 2026D85 and 2026D86

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add 2 new phase2 scenarios 2026D85 and 2026D86. This contains new definition of MTD (Fabio, Marta), HGCal and support structure for GE0

#### PR validation:

Tested with 2 new workflows: 38234.0 and 38634.0

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

File: Configuration/Geometry/README.md [hunk 1]
```diff
@@ -94,11 +99,13 @@ Several detector combinations have been generated:
...
+* D77 = T24+C14+M9+I13+O7+F6 (Current default scenario)
 * D78 = T22+C14+M9+I13+O7+F6
 * D79 = T23+C14+M9+I13+O7+F6
 * D80 = T25+C14+M9+I13+O7+F6
 * D81 = T26+C14+M9+I13+O7+F6
 * D82 = T21+C15+M9+I13+O7+F7
 * D83 = T24+C16+M9+I13+O7+F6
 * D84 = T24+C13+M7+I11+O6+F6 (For HGCAL study on evolution of HGCal replacing D70)
+* D85 = T24+C14+M9+I14+O7+F6
+* D86 = T24+C17+M10+I14+O8+F6
```
Comment (mmusich) [inline Configuration/Geometry/README.md:111]: This will conflict with the geometry introduced at https://github.com/cms-sw/cmssw/pull/34120.
@adewit FYI

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/README.md (hunk 1 above)
Comment (bsunanda) [reply Configuration/Geometry/README.md:111]: I understand this only now. I did not realize that T27 is put in till I submitted this PR. If #34120 is merged, I shall take proper action

**Comment by fabiocos** [comment]
The updated description of material in MTD scenario I14 as implemented by @martatornago has been presented in https://indico.cern.ch/event/1057473/contributions/4444289/attachments/2279146/3872237/ETLGeometryUpdate.pdf

File: Configuration/Geometry/README.md [hunk 2]
```diff
@@ -62,25 +62,30 @@ Calorimeters:
...
 * M8: same as M7 with changed number of strips for GE21
 * M9: same as M8 with GE0 replacing ME0
+* M10: same as M9 but with a realistic support structure for GE0
 
 Fast Timing system:
 * I10: Fast Timing detector (LYSO barrel (bars along phi flat), silicon endcap), w/ passive materials, ETL in position defined in O4, material adjustments
 * I11: Same as I10, xml reorganized, comparison base for new ETL and DD4hep migration
 * I12: Starting from I11, new ETL layout from MTD TDR
 * I13: Starting from I11, new ETL layout from post MTD TDR (2 sectors per disc face)
+* I14: Same as I13, updated sensor structure, disc z location and passive ,aterials
```
Comment (cvuosalo) [inline Configuration/Geometry/README.md:80]:
```suggestion
* I14: Same as I13, updated sensor structure, disc z location and passive materials
```

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@chayanit @srimanob Can you approve this PR urgently?

**Comment by kskovpen** [comment]
+1

**Comment by srimanob** [comment]
@adewit @mmusich 
Would you mind to update https://github.com/cms-sw/cmssw/pull/34120 to D87? Thanks in advance.

File: Configuration/Geometry/README.md [hunk 3]
```diff
@@ -94,11 +99,13 @@ Several detector combinations have been generated:
 * D68 = T21+C11+M6+I11+O5+F4 (For HGCAL study on evolution of detector)
 * D70 = T21+C13+M7+I11+O6+F6 (For HGCAL study on evolution of detector)
 * D76 = T21+C14+M9+I13+O7+F6
-* D77 = T24+C14+M9+I13+O7+F6
+* D77 = T24+C14+M9+I13+O7+F6 (Current default scenario)
```
Comment (srimanob) [inline Configuration/Geometry/README.md:102]: What do you mean by the current default scenario? 
The default scenario is D76. 

Of course, we can try to move to D77 where T24==T21 but DD4hep support.

File: Configuration/Geometry/README.md (hunk 1 above)
Comment (srimanob) [reply Configuration/Geometry/README.md:111]: For D86, do you want to jump both Calorimeter and Muon, not one-by-one as we usually do?

File: Configuration/Geometry/README.md (hunk 3 above)
Comment (bsunanda) [reply Configuration/Geometry/README.md:102]: It cannot be. D77 is identical to D76. While D77 can run with dd4hep, D76 does not.

**Comment by bsunanda** [comment]
Also C17, M10 and O8 go hand in hand. An improved muon version will come soon afterwards.

File: Configuration/Geometry/README.md (hunk 3 above)
Comment (srimanob) [reply Configuration/Geometry/README.md:102]: I understand. I just say that the current geometry we used for relvals and short matrix test is D76.

**Comment by srimanob** [comment]
assign @cms-sw/mtd-dpg-l2

**Comment by srimanob** [comment]
assign mtd-dpg

**Comment by srimanob** [comment]
+Upgrade

Note that I would like to get confirmation on moving from Tracker also as D85 are on both PRs.

**Comment by bsunanda** [comment]
@fabiocos can you approve this PR?

**Comment by fabiocos** [comment]
+1

the MTD update was provided by the DPG based on the updates by @martatornago , from the tests performed no overlap is observed in scenario D85 . A corresponding update of ```Validation/Geometry``` will be proposed to exploit this PR. This will require an update of the material ```Borathed Polyetyl.```.

**Comment by fabiocos** [comment]
+mtd-dpg

**Comment by fabiocos** [comment]
@silviodonato @qliphy @perrotta please check operations as well, my +1 approved (involuntarily) also that, although it looks ok to me (I had tried it already in my private test https://github.com/fabiocos/cmssw/tree/fc-etlv6 )

**Comment by qliphy** [comment]
+1

**Pull request merged**